attributes/activiti-admin.properties.rb

```diff
@@ -2,9 +2,9 @@
 default['aps-core']['activiti-admin-properties']['security.rememberme.key'] = 'activitis3cr3tk3y'
 
 # Database config
-default['aps-core']['activiti-admin-properties']['datasource.driver'] = 'com.mysql.jdbc.Driver' if node['aps-core']['admin_db']['engine'] == 'mysql'
-default['aps-core']['activiti-admin-properties']['datasource.driver'] = 'org.postgresql.Driver' if node['aps-core']['admin_db']['engine'] == 'postgres'
-default['aps-core']['activiti-admin-properties']['datasource.driver'] = 'org.h2.Driver' if node['aps-core']['admin_db']['engine'] == 'h2'
+default['aps-core']['activiti-admin-properties']['datasource.driver'] = lazy { 'com.mysql.jdbc.Driver' } if node['aps-core']['admin_db']['engine'] == 'mysql'
```

Comment:
why do you need lazy evaluation here?

Reply:
Because the converge didnt work without it